community/values/src/main/java/org/neo4j/values/storable/PointValue.java

```diff
@@ -280,9 +280,42 @@ public static PointValue fromMap( MapValue map )
     private static Pattern quotesPattern = Pattern.compile( "^[\"']|[\"']$" );
 
     public static PointValue parse( CharSequence text )
+    {
+        return PointValue.parse( text, null );
+    }
+
+    /**
+     * Parses the given text into a PointValue. The information stated in the header is saved into the PointValue
+     * unless it overriden by the information in the text
+     * @param text the input text to be parsed into a PointValue
+     * @param fieldsFromHeader must be a value obtained from {@link #parseIntoArray(CharSequence)} or null
+     * @return a PointValue instance with information from the {@param fieldsFromHeader} and {@param text}
+     */
+    public static PointValue parse( CharSequence text, AnyValue[] fieldsFromHeader )
```

Comment:
<Repeating my comment from the outdated diff>
How about renaming fieldsFromHeader to something like defaultFields to make it user agnostic?